types/types.go

```diff
@@ -568,6 +568,8 @@ type SystemContext struct {
 	DockerDisableDestSchema1MIMETypes bool
 	// Directory to use for OSTree temporary files
 	OSTreeTmpDirPath string
+	// Allow uncompressed image layers in Docker images
+	DockerAcceptUncompressedLayers bool
```

Comment:
```suggestion
	// Allow uncompressed image layers in Docker images
	DockerAcceptUncompressedLayers bool
	
	// === ostree.Transport overrides ===
	// Directory to use for OSTree temporary files
	OSTreeTmpDirPath string
```

I should have cleaned up that part long ago.